client/src/components/ui/score-card.tsx

```diff
@@ -0,0 +1,27 @@
+export type Score = "high" | "medium" | "low";
+
+interface ScoreIndicatorProps {
+  score: Score;
+  className?: string;
+  children?: React.ReactNode;
+}
+
+export const ScoreIndicator = ({
+  score,
+  children,
+  className = "",
+}: ScoreIndicatorProps) => {
+  const bgColorClass = {
+    high: "bg-high",
+    medium: "bg-medium",
+    low: "bg-low",
+  }[score];
```

Comment:
This dictionary has to come from a property of the component and be the default one, but allowing using a different one when needed.

Reply:
Can you please give an example to help me understand it better, how would that look like ? @andresgnlez